.github/workflows/native-build-development.yml

```diff
@@ -9,9 +9,27 @@ on:
   workflow_dispatch:
 
 jobs:
-  build_with_native:
+  generate-json-matrix:
+    name: Native Tests - Read JSON matrix
     runs-on: ubuntu-latest
     if: "github.repository == 'quarkusio/quarkus-quickstarts' || github.event_name == 'workflow_dispatch'"
+    outputs:
+      matrix: ${{ steps.generate.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: generate
+        run: |
+          json=$(./.github/generate-native-matrix.sh | tr -d '\n')
```

Comment:
I don't think the leading `./` is required here

Reply:
Yeah, removed.